content/docs/rendering-elements.md

```diff
@@ -1,75 +1,77 @@
 ---
 id: rendering-elements
-title: Rendering Elements
+title: Rendering av element
 permalink: docs/rendering-elements.html
 redirect_from:
   - "docs/displaying-data.html"
 prev: introducing-jsx.html
 next: components-and-props.html
 ---
 
-Elements are the smallest building blocks of React apps.
+Element är de minsta byggstenarna i React-applikationer.
 
-An element describes what you want to see on the screen:
+Ett element beskriver vad som skall visas på skärmen:
 
 ```js
 const element = <h1>Hello, world</h1>;
 ```
 
-Unlike browser DOM elements, React elements are plain objects, and are cheap to create. React DOM takes care of updating the DOM to match the React elements.
+Till skillnad från DOM-elementen i en webbläsare är React-element enkla och kostnadseffektiva objekt. React DOM tar hand om uppdatering av DOM och tillhörande React-element.
 
->**Note:**
+
+>**Obs:**
 >
->One might confuse elements with a more widely known concept of "components". We will introduce components in the [next section](/docs/components-and-props.html). Elements are what components are "made of", and we encourage you to read this section before jumping ahead.
+>Man kan förväxla element med det mer kända begreppet "komponenter". Vi kommer att introducera komponenter i [nästa avsnitt](/docs/components-and-props.html). Elementen utgör grunden för komponenterna, så vi rekommenderar dig att läsa detta avsnitt innan du går vidare.
 
-## Rendering an Element into the DOM {#rendering-an-element-into-the-dom}
+## Rendering av element i DOM {#rendering-an-element-into-the-dom}
 
-Let's say there is a `<div>` somewhere in your HTML file:
+Låt oss säga att du har en `<div>` någonstans i din HTML-fil:
 
 ```html
 <div id="root"></div>
 ```
 
-We call this a "root" DOM node because everything inside it will be managed by React DOM.
+Vi kallar detta en "root" DOM-nod eftersom allt inom den kommer att hanteras av React DOM.
```

Comment:
'"root" DOM node' could be translated to '"root"-DOM-nod' here.

Reply:
I see this one occurring in at least two more places, but I didn't leave a comment there.

Reply:
Fixing. 👍